Content.Server/GameObjects/Components/Interactable/Tools/CrowbarComponent.cs

```diff
@@ -43,6 +43,10 @@ public void AfterAttack(AfterAttackEventArgs eventArgs)
                 var underplating = _tileDefinitionManager["underplating"];
                 mapGrid.SetTile(eventArgs.ClickLocation, new Tile(underplating.TileId));
                _entitySystemManager.GetEntitySystem<AudioSystem>().Play("/Audio/items/crowbar.ogg", Owner);
+                //Actually spawn the relevant tile item at the right position and give it some offset to the corner.
+                var tileItem = Owner.EntityManager.SpawnEntity(tileDef.ItemDropPrototypeName);
+                tileItem.Transform.GridPosition = coordinates;
+                tileItem.Transform.WorldPosition += new Robust.Shared.Maths.Vector2(0.2f, 0.2f);
```

Comment:
You can just do this here:

```suggestion
                tileItem.Transform.WorldPosition += (0.2f, 0.2f);
```